.github/workflows/ci.yml

```diff
@@ -58,6 +58,26 @@ jobs:
         bundle install --jobs 4 --retry 3
         bundle exec rubocop
         bundle exec erblint **/*.html.erb
+  test:
+    needs: lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.2
+    - uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: gems-build-rails-main-ruby-2.7.2-${{ hashFiles('**/Gemfile.lock') }}
+    - name: Run benchmarks
+      run: |
+        bundle config path vendor/bundle
+        bundle update
+        bundle exec rake benchmark
+        bundle exec rake translatable_benchmark
+        bundle exec rake slotable_benchmark
```

Comment:
Non-blocking, but It would be awesome if we could expose the results of these benchmarks as a build artifact.

Reply:
It would be! I'd also like to see us put in a basic stopgap for failing a build if they change too drastically.